kratos/utilities/function_parser_utility.cpp

```diff
@@ -0,0 +1,211 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//  Collaborator:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+#include "tinyexpr/tinyexpr/tinyexpr.h"
+
+// Project includes
+#include "utilities/function_parser_utility.h"
+#include "utilities/string_utilities.h"
+
+namespace Kratos
+{
+
+BasicGenericFunctionUtility::BasicGenericFunctionUtility(const std::string& rFunctionBody)
+    : mFunctionBody(rFunctionBody)
+{
+    // Correcting function from python style to tinyexpr
+    // NOTE: tinyexpr does not support capital letters
```

Comment:
```suggestion
```

this is no longer true :tada:

Reply:
Yes I updated it, but I forgot to clean up comment